firestore/testapp/Assets/Firebase/Editor/Builder.cs

```diff
@@ -43,6 +43,23 @@ public class Builder {
     Build(options);
   }
 
+  public static void BuildTvos() {
```

Comment:
nit: it looks a bit wired with the capital choice, same to the previous Ios, but I don't know what's the best way to deal with it :(

Reply:
Yeah, I'm not a huge fan either. I looked it up and I _think_ this is actually in line with [Google's C# style guide](https://google.github.io/styleguide/csharp-style.html). Methods are Pascal Case and only acronyms of 2 letters or less are fully capitalized.

Reply:
It still makes my eyes hurt, though. :)